### PR TITLE
Functionality

### PR DESCRIPTION
**Pull request by Theodoraldo**
At this juncture, I :

- [ ] Handle events in a React app.
- [ ] Use styles in a React app.
- [ ] Use store, actions and reducers in React.
- [ ] Write integration tests with a React testing library.
- [ ] Connect React and Redux.
- [ ] Use React Router.
- [ ] Use React props.
- [ ] Use React components.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,174 @@
...
+
+### Key Features <a name="key-features"></a>
+
+- **Show countries information**
+- **Displays continent information**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo](soon)
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly ensure you deploy your app and provide its link in the README.

  image

- [ ] Kindly record a video presentation of your app and include the link in the README. Ensure you outline the piece of code you are proud of or find interesting.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,174 @@
...
+```
+-
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
```
Comment (TedLivist) [inline README.md:126]: - [ ] One of your tests is failing. Kindly fix that.

  image

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+const continentList = [
+  { id: 1, continent: 'Africa', map: afirca },
+  { id: 2, continent: 'America', map: america },
+  { id: 3, continent: 'Antarctic', map: antarctic },
+  { id: 4, continent: 'Asia', map: asia },
+  { id: 5, continent: 'Europe', map: europe },
+  { id: 6, continent: 'Oceania', map: oceania },
+];
+
+export default function Home() {
```
Comment (TedLivist) [inline src/components/Home.js:21]: - [ ] Your Homepage should have a search or filtering feature that allows users to filter items on the homepage. Kindly implement that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Theodoraldo,

Good job so far!
✔️ Implementation of unique routing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+              </div>
+              <div className="cont-body">
+                <p className="cont-name">{continent.continent}</p>
+                <img src={continent.map} alt="continent logo" width={200} />
+              </div>
+            </div>
+          ))
+        }
+      </div>
+    </>
```
Comment (CollinsTatang) [inline src/components/Home.js]: - [ ] Your search box is still not working as mentioned by the previous reviewer on the homepage.
 - Please user should be able to filter or search for a particular continent using the search box on the homepage.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Theodoraldo , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your app was deployed successfully. ✔️ 
- All your app test pass successfully. ✔️ 

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by osamaashraf6 — APPROVED** [review]
Hi @Theodoraldo 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (3)

## Highlights🎉🎉

- [ ] Use the redux for the state management💯
- [ ] All tests passed 👍
- [ ] Has a live demo for the project🚀

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**